docs/Auth-with-Ionic2.md

```diff
@@ -142,7 +142,7 @@ and add the following three entries.
 
 your `app.module.ts` file should look something like this.
```

Comment:
This section needs to be tailored to follow this guide: https://firebase.google.com/docs/auth/web/cordova